packages/logs/src/boot/preStartLogs.spec.ts

```diff
@@ -217,14 +217,14 @@ describe('preStartLogs', () => {
 
     describe('basic methods instrumentation', () => {
       it('should instrument fetch even if tracking consent is not granted', () => {
-        const originalFetch = window.fetch
-
-        strategy.init({
-          ...DEFAULT_INIT_CONFIGURATION,
-          trackingConsent: TrackingConsent.NOT_GRANTED,
-        })
-
-        expect(window.fetch).not.toBe(originalFetch)
+        expect(
+          instrumentationIsAddedTo(window, 'fetch', () => {
```

Comment:
I agree with the problem! I felt like `didAddInstrumentationTo()` could still be read in the unintended way, though, so I went a bit further and switched to a fluent interface. The new API follows this pattern:
```ts
callbackAddsInstrumentation(() => {
  // ... code that's intended to instrument window.foo ...
}).toMethod(window, 'foo').whenCalled()
```
More verbose, but hopefully this will ultimately be clearer.